routes/toiletsRouter.js

```diff
@@ -4,11 +4,9 @@ const router = express.Router();
 const {
   getNearToiletsList,
   getReviewsList,
-  postSOS,
 } = require("../controller/toiletsController");
 
 router.get("/", getNearToiletsList);
 router.get("/review/:id", getReviewsList);
```

Comment:
```suggestion
router.get("/review/:toiletId", getReviewsList);
```
라우터 params 부분은 다같이 리펙토링 진행 해봐야 할것 같아요!

Reply:
넵 칸반카드에 추가...